mods/ts/rules/aircraft.yaml

```diff
@@ -238,6 +241,7 @@ SCRIN:
 		ROT: 3
 		Speed: 168
 		AirborneUpgrades: airborne
+		MoveIntoShroud: false
```

Comment:
Did the original banshee have the FlyCircle behaviour that ours does?  That negates the effectiveness of MoveIntoShroud a fair bit.